tests/baselines/reference/nodeModulesConditionalPackageExports(module=node16).errors.txt

```diff
@@ -81,8 +84,14 @@ tests/cases/conformance/node/node_modules/inner/index.d.ts(2,13): error TS2303:
     import * as ts from "inner/types";
     export { cjs };
     export { mjs };
+             ~~~
+!!! error TS2303: Circular definition of import alias 'mjs'.
     export { type };
+             ~~~~
+!!! error TS2303: Circular definition of import alias 'type'.
     export { ts };
+             ~~
+!!! error TS2303: Circular definition of import alias 'ts'.

```

Comment:
I'm not sure what’s going on with these, but I think it’s my mistake. Nothing should be different about what happens in a Node ESM file.